src/main/java/edu/cornell/kfs/paymentworks/batch/PaymentWorksUploadSuppliersStep.java

```diff
@@ -0,0 +1,309 @@
+					.createPaymentworksSupplierUploadList(results);
+
+			// upload list of vendors
+			boolean uploaded = paymentWorksWebService.paymentworksUploadSuppliers(paymentworksSupplierUploadList);
+
+			if (uploaded) {
+				// loop through list and upload
+				for (PaymentWorksVendor newVendor : results) {
+
+					String supplierStatusType = !StringUtils.startsWith(newVendor.getVendorRequestId(), "KFS")
+							? PaymentWorksConstants.SupplierUploadSummaryTypes.KFS_NEW_VENDORS
+							: PaymentWorksConstants.SupplierUploadSummaryTypes.PMW_NEW_VENDORS;
+
+					boolean updateStatus = !StringUtils.startsWith(newVendor.getVendorRequestId(), "KFS") ? true
+							: false;
+
+					processVendor(newVendor, PaymentWorksConstants.PaymentWorksNewVendorStatus.PROCESSED,
+							PaymentWorksConstants.PaymentWorksStatusText.PROCESSED, PaymentWorksConstants.ProcessStatus.SUPPLIER_UPLOADED,
+							supplierStatusType, updateStatus);
+				}
+			}
+		}
+	}
+
+	public void uploadVendorUpdateApprovedSupplierFile() {
+
+		// get list of approved vendors
+		Collection<PaymentWorksVendor> results = paymentWorksVendorService.getPaymentworksVendorRecords(
+				PaymentWorksConstants.ProcessStatus.VENDOR_APPROVED, null, PaymentWorksConstants.TransactionType.VENDOR_UPDATE);
+
+		if (ObjectUtils.isNotNull(results) && !results.isEmpty()) {
+			// convert to supplier upload DTO
+			List<PaymentWorksSupplierUploadDTO> paymentworksSupplierUploadList = new PaymentWorksSupplierConversionUtil()
+					.createPaymentworksSupplierUploadList(results);
+
+			// upload list of vendors
+			boolean uploaded = paymentWorksWebService.paymentworksUploadSuppliers(paymentworksSupplierUploadList);
+
+			if (uploaded) {
+				// loop through list and upload
+				for (PaymentWorksVendor newVendor : results) {
+
+					processVendor(newVendor, PaymentWorksConstants.PaymentWorksUpdateStatus.PROCESSED,
+							PaymentWorksConstants.PaymentWorksStatusText.PROCESSED, PaymentWorksConstants.ProcessStatus.SUPPLIER_UPLOADED,
+							PaymentWorksConstants.SupplierUploadSummaryTypes.VENDOR_UPDATES, false);
+				}
+			}
+		}
+	}
+
+	public void updateNewVendorDisapprovedStatus() {
+
+		// get list of disapproved vendors
+		Collection<PaymentWorksVendor> results = paymentWorksVendorService.getPaymentworksVendorRecords(
+				PaymentWorksConstants.ProcessStatus.VENDOR_DISAPPROVED, PaymentWorksConstants.PaymentWorksStatusText.APPROVED,
+				PaymentWorksConstants.TransactionType.NEW_VENDOR);
+
+		// loop through list and set status to rejected
+		for (PaymentWorksVendor newVendor : results) {
+
+			processVendor(newVendor, PaymentWorksConstants.PaymentWorksNewVendorStatus.REJECTED,
+					PaymentWorksConstants.PaymentWorksStatusText.REJECTED, PaymentWorksConstants.ProcessStatus.VENDOR_DISAPPROVED,
+					PaymentWorksConstants.SupplierUploadSummaryTypes.DISAPPROVED_VENDORS, true);
+		}
+	}
+
+	protected void processVendor(PaymentWorksVendor paymentworksNewVendor, String requestStatus,
```

Comment:
There are multiple variables and method parameters where the name is not camelcased properly. I would change payementworksNewVendor to paymentWorksNewVendor here and do similar camelcasing elsewhere.